src/libs/NumberFormatUtils.ts

```diff
@@ -0,0 +1,9 @@
+function format(locale: string, number: number, options?: Intl.NumberFormatOptions) {
+    return new Intl.NumberFormat(locale, options).format(number);
+}
+
+function formatToParts(locale: string, number: number, options?: Intl.NumberFormatOptions) {
+    return new Intl.NumberFormat(locale, options).formatToParts(number);
+}
```

Comment:
@BartoszGrajdek Could you add return types for these functions?

Reply:
Should be ok now @fabioh8010